packages/checkbox-react/src/CheckboxGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Checkbox } from "./Checkbox";
+
+interface Props {
+    legend: string;
+    choices: string[];
```

Comment:
We might want to be able to pass both name and label value for each option/choice. Maybe something like
```tsx
interface Choice {
    name: string;
    label: string;
}
interface Props {
    /* ... */
    choices: Choice[] | string[];
}
```